apps/full-site-editing/full-site-editing-plugin/premium-content/blocks/button/color-edit.js

```diff
@@ -0,0 +1,211 @@
+// The code in this file is copied entirely from https://github.com/WordPress/gutenberg/blob/master/packages/block-library/src/button/color-edit.js
```

Comment:
Code copied from Gutenberg, feel free to skip it during review.

Reply:
Maybe leave an issue for follow up? This feels like something we could try exporting as a npm package from the core pacakge if there's general reuse value here.

Reply:
Just found out there is a plan for removing these files in favor of using the new `color` and `style` support flags. It was actually done in https://github.com/WordPress/gutenberg/pull/21266 but reverted in https://github.com/WordPress/gutenberg/pull/21923 due to regressions. 

Will investigate if we can use those support flags to reduce code here.

Reply:
Was able to get rid of those duplicate files in favor of using [the `color` hooks](https://github.com/WordPress/gutenberg/blob/master/packages/block-editor/src/hooks/color.js) in 750d3d8 🎉